index.html

```diff
@@ -10,9 +10,18 @@
 </head>
 <body>
   <header class="navbar">
-  <nav class="container">
+    <div class="nav-logo">
     <a  class="logo" href="#">My logo</a>
-    <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+  </div>
+  <nav class="container">
+    <div class="menu-wrapper">
+      <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+    </div>
+    <ul class="nav-list">
+      <li class="nav-item">Portfolio</li>
+      <li class="nav-item">about</li>
+      <li class="nav-item">contact</li>
+    </ul>
```

Comment:
- [ ] Kindly use the `a` tag for the menu as it is the appropriate tag for links.